packages/MarkdownEditor-Core.package/MarkdownCommentBlock.class/instance/commentStart.st

```diff
@@ -1,4 +1,4 @@
 accessing
 commentStart
 
-	^ self firstLine findString: '<!--'
\ No newline at end of file
+	^ self firstLine findString: self class openingTag
```

Comment:
We could consider adding an accessor for the opening and closing tag on the object side too which would just call the class accessor. That way calling it on the object side would get a little cleaner but I'm not sure if that is necessary.